### ISSUE TITLE
no-unresolved for react native

### ISSUE THREAD

**Issue by damusnet** [Intellicode/eslint-plugin-react-native#32]
Basically, could [the settings in this issue](https://github.com/benmosher/eslint-plugin-import/issues/279#issuecomment-215052176) be included in the `eslint-plugin-react-native`?

```json
  "settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js", ".ios.js", ".android.js"]
      }
    }
  }
```

**Comment by Intellicode** [comment]
@damusnet That sounds useful, let me look into it!

**Comment by bbrock25** [comment]
additionally, is there a way to have eslint pick up on the [Static Image Resources](https://facebook.github.io/react-native/docs/images.html#static-image-resources)

ie: files that are named as follows: 
`"file-name@1x.png", "file-name@2x.png", "file-name@3x.png"` 
but referenced as: `require("../images/file-name.png")`

**Comment by xareelee** [comment]
Hi, I encounter a problem with `require()` for images.

```js
const flagImg = require('./img/Login_flag_1.png');
```

and ESLint reports the error (import/no-unresolved):

```
  12:25  error  Unable to resolve path to module './img/Login_flag_1.png'               import/no-unresolved
```

How do I do to resolve this problem?

**Comment by gustavjf** [comment]
> additionally, is there a way to have eslint pick up on the Static Image Resources

@bbrock25 Have you found a solution for this?

**Comment by bbrock25** [comment]
@gustavjf I haven't been working in RN lately, I'm pretty sure we ended up ignoring those imports :(

**Comment by pcguru** [comment]
This plugin works for me: https://www.npmjs.com/package/eslint-import-resolver-react-native
Linked from here: https://github.com/benmosher/eslint-plugin-import/wiki/Resolvers

**Comment by sturmenta** [comment]
For using react-native-web, this works good 🔥🔥🔥

```
 "import/no-unresolved": [
      2,
      {
        "ignore": [
          "react-native"
        ]
      }
    ],
```
